### DISCUSSION TITLE
Add docker stdout to deploy logs

### DISCUSSION THREAD

**Discussion by grahamhealy20** [coollabsio/coolify#1998] [Feature Requests]
Hi there,

I'm getting started with Coolify (loving it so far by the way!) and trying to deploy a Python nixpack. I know the deploy will fail as I haven't added all of the required env vars, but I wish the deployment log would provide me more information about _why_ the deployment is unhealthy/failed. Currently it logs out something like this:

```[2024-Apr-15 13:29:05.006613] Attempt 1 of 10 | Healthcheck status: "starting"
[2024-Apr-15 13:29:10.112174]

COMMAND: docker inspect --format='{{json .State.Health.Status}}' dso8ogg-132801577302

OUTPUT :
"unhealthy"


[2024-Apr-15 13:29:10.118074] Attempt 2 of 10 | Healthcheck status: "unhealthy"
[2024-Apr-15 13:29:10.123310] Removing old containers.
[2024-Apr-15 13:29:10.128265] New container is not healthy, rolling back to the old container.
[2024-Apr-15 13:29:10.258450] Rolling update completed.
```

But I would love to the see the actual container stdout also. I tried manually looking for the docker container logs via the command line, but the deploy has cleaned the container up post deploy failure, so I can't retrieve them. The only way I was able to get access to the logs was by disabling health checks temporarily, checking the logs, then re-enabling. Is this currently possible, or something that we could think about implementing, maybe included in the debug logs? 

Thanks again!

**Comment by nightofday** [comment]
experienced the same issue here.
